packages/pangea-sdk/.sdk-ci.yml

```diff
@@ -80,72 +84,71 @@
         ENV: ${SERVICE_VAULT_ENV}
         TEST: test_vault
   rules:
-    - if: '$CLOUD == "GCP" && $TEST == "test_file_scan"'
+    - if: $CI_COMMIT_BRANCH && '$CLOUD == "GCP" && $TEST == "test_file_scan"'
```

Comment:
Why do we need to add this condition?

Reply:
Don't wanna run the long integration tests on tag push (because no code is changing, the tests would already be running on the push of the version bump commit).